isodatetime/data.py

```diff
@@ -1390,7 +1385,7 @@ def _add_months(self, num_months):
             if self.get_is_week_date():
                 was_week_date = True
             self.to_calendar_date()
-        for i in range(abs(num_months)):
+        for _ in range(abs(num_months)):
```

Comment:
I get why PyLint suggests this, but I find it less readable/intuitive. (Oh well)